src/App.js

```diff
@@ -7,6 +7,7 @@ import { TranslationsPageWithRouter } from './translations/TranslationsPage';
 import ContributionsPage from './contributions/ContributionsPage';
 import ThemePage from './theme/ThemePage';
 import AuthUtils from './utils/AuthUtils';
+import ApiUtils from "../utils/ApiUtils";
```

Comment:
Should this be `import ApiUtils from "./utils/ApiUtils";` ?

Reply:
Fixed, thank you!